client/mysqldump.c

```diff
@@ -742,6 +746,14 @@ static void write_header(FILE *sql_file, char *db_name)
             "/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;\n",
             path?"":"NO_AUTO_VALUE_ON_ZERO",compatible_mode_normal_str[0]==0?"":",",
             compatible_mode_normal_str);
+    fprintf(sql_file,
+            "/*!50601 SELECT count(*) INTO @is_rocksdb_supported FROM"
```

Comment:
Another perspective here, as I understand, the purpose of these is in the case that the result of this dump it attempted to be imported/replayed against an older server version. For the purposes of MyRocks, we _know_ that there will never be a server version older then 5.6 with the plugin installed, not from us, not from Facebook, not from MariaDB. Even if we did see one, not setting this bulk load option is not a catastrophic event, it is simply an optional optimization. So I'm thinking this is OK as is, otherwise, (since I am not terribly familiar with mysqldump history and semantics) I am not sure how these should be treated. Simply change the version requirement to some 5.1? Remove them entirely?